FileAPI/blob/Blob-slice.html

```diff
@@ -35,7 +35,24 @@
     type: "content/type",
     desc: "blob2."
   });
+  
```

Comment:
trailing whitespace on this line needs to be removed https://travis-ci.org/w3c/web-platform-tests/jobs/176824939